ckan/plugins/interfaces.py

```diff
@@ -892,6 +892,11 @@ def get_actions(self):
         By decorating a function with the `ckan.logic.side_effect_free`
         decorator, the associated action will be made available by a GET
         request (as well as the usual POST request) through the action API.
+
+        By decrorating a function with the 'ckan.plugins.toolkit.chained_action,
+        the action will be chained to another function defined in plugins with a
+        "first plugin wins" pattern, which means the first plugin declaring a
+        chained action should be called first.
```

Comment:
^ I meant to say I would add something like this paragraph to the one you've written.